cmd/mgo/main.go

```diff
@@ -40,7 +40,7 @@ func initServices() (model.AppStorage, model.UserStorage, model.TokenStorage, mo
 func initRouter() model.Router {
 	appStorage, userStorage, tokenStorage, tokenService := initServices()
 
-	router, err := ihttp.NewRouter(nil, appStorage, userStorage, tokenStorage, tokenService)
+	router, err := ihttp.NewRouter(nil, appStorage, userStorage, tokenStorage, tokenService, ihttp.ServeDefaultStaticPages())
```

Comment:
this pages should be served without any additional settings, because they are required